web/init/src/components/shared/NavBar.jsx

```diff
@@ -126,7 +126,7 @@ export class NavBar extends React.Component {
             imageLoaded: true,
           })
         })
-        .catch(() => this.setState({ updatedState }))
+        .catch(() => this.setState({ ...updatedState, imageLoaded: true }))
```

Comment:
This is fine but it will completely disable retries.

Thanks for catching my bug here too 😄